expat/doc/reference.html

```diff
@@ -292,7 +292,7 @@ <h2><a name="overview">Overview</a></h2>
 #include &lt;expat.h&gt;
 
 #ifndef XMLCALL
-#if defined(_MSC_EXTENSIONS) &amp;&amp; !defined(__BEOS__) &amp;&amp; !defined(__CYGWIN__)
+#if defined(_MSC_VER) &amp;&amp; !defined(__BEOS__) &amp;&amp; !defined(__CYGWIN__)
```

Comment:
The commit message talks about `__declspec(dllimport)`, but this section of the HTML is actually about something completely different (cdecl), right?